cs221bot.py

```diff
@@ -190,9 +188,9 @@ async def on_command_error(ctx, error):
 
         # prints full traceback
         try:
-            await ctx.send("```" + "".join(traceback.format_exception(etype, error, trace, 999)) + "```".replace("C:\\Users\\William\\anaconda3\\lib\\site-packages\\", "").replace("D:\\my file of stuff\\cs221bot\\", ""))
+            await ctx.send(("```" + "".join(traceback.format_exception(etype, error, trace, 999)) + "```").replace("C:\\Users\\William\\anaconda3\\lib\\site-packages\\", "").replace("D:\\my file of stuff\\cs221bot\\", ""))
```

Comment:
These are the lines where I added the parenthesis. Without them, the replace only acts on ``` rather than the entire message.